src/components/FeatureViews/Registration/Registration.js

```diff
@@ -1,14 +1,14 @@
 import "../../../theme/theme.scss";
 import React from "react";
 import RegistrationLanding from "./RegistrationLanding";
+import Zoom from "@material-ui/core/Zoom";
 
-const Registration = () => (
-    <>
-        {/*<Paper className="paper">*/}
-        {/*    <RegistrationUserActions />*/}
-        {/*</Paper>*/}
-        <RegistrationLanding />
-    </>
-);
+const Registration = () => {
+    return <div>
+        <Zoom>
+            <RegistrationLanding />
+        </Zoom>
+    </div>;
```

Comment:
Is the `div` necessary here?

Reply:
yeah, it gets buggy without it because it's trying to render an undefined component.